core/src/test/scala/unit/kafka/utils/TestUtils.scala

```diff
@@ -966,6 +966,49 @@ object TestUtils extends Logging {
     }.toMap
   }
 
+  def waitUntilTopicPresent(client: Admin, topic: String, maxAttempts: Int = 3): Unit = {
+    if (maxAttempts <= 0) {
+      throw new IllegalArgumentException("0 or negative maxAttempts")
+    }
+
+    var remaining = maxAttempts
+    while (remaining > 0) {
```

Comment:
Maybe reuse the TestUtils.waitUntilTrue instead of creating the while loop?

Reply:
Rewrote it with the waitUntilTrue